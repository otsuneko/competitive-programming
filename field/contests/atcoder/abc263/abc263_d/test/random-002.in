7 8 5
2 3 10 5 6 1 1
