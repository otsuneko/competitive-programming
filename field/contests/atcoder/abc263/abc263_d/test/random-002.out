[2, 3, 10, 5, 6, 1, 1]
28
